helm/dagster/schema/schema/charts/dagster/subschema/retention.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import BaseModel, Extra
+
+
+class TickRetentionByType(BaseModel):
+    skipped: Optional[int]
+    success: Optional[int]
+    failure: Optional[int]
+    started: Optional[int]
```

Comment:
Technically, these are not optional, since you've put default values in `values.yaml`.
```suggestion
    skipped: int
    success: int
    failure: int
    started: int
```